source/userguide/kb.rst

```diff
@@ -726,26 +726,33 @@ To override MindMeld's default QuestionAnswerer configuration with custom settin
+      }
   }
 
-The ``model_type`` can be one of 'keyword' or 'text'. While specifying the former optimizes your QuestionAnswerer search for keywords or short spans of text, the latter optimizes for searching on larger paragraphs or passages of unstructured text. In order to leverage embeddings based semantic matching along with surface-level text features, you can specify one of three model types- 'embedder', 'embedder_text' and 'embedder_keyword'. For leveraging appropriate embeddings, you can specify embedder configurations in the :data:`QUESTION_ANSWERER_CONFIG` with an additional key 'model_settings'. For full details on using embedders in QuestionAnswerer, check the :ref:`Leveraging semantic embeddings <semantic_embeddings>` section below. Note that the specified ``model_type`` is also the default ``query_type`` for all QuestionAnswerer calls in your application, but you can always pass in a different query type to your ``qa.get()`` command as desired.
+The ``query_type`` can be one of ``keyword`` or ``text``. While specifying the former optimizes your QuestionAnswerer search for keywords or short spans of text, the latter optimizes for searching on larger paragraphs or passages of unstructured text. In order to leverage embeddings-based semantic matching along with surface-level text features matching, you can specify one of the three embedder query types- ``embedder``, ``embedder_text``, or ``embedder_keyword``. For using embeddings of your choice, you can specify embedder configurations in the :data:`QUESTION_ANSWERER_CONFIG` within the key 'model_settings'. For full details about using embedders in QuestionAnswerer, check the :ref:`Leveraging semantic embeddings <semantic_embeddings>` section below. Note that the specified ``query_type`` is also the default ``query_type`` for all QuestionAnswerer calls in your application, but you can always pass in a different query type to your ``qa.get(query_type=...)`` command as desired.
 
 .. _non_elasticsearch_question_answerer:
 
-QuestionAnswerer without Elasticsearch backend
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+QuestionAnswerer without Elasticsearch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Search operations of QuestionAnswerer are by default backed by Elasticsearch. While it is recommended to keep using Elasticsearch backend for QuestionAnswerer, it might not always be feasible to use it due to some or the other constraints. In such cases, one can easily disable Elasticsearch usage by setting a disable-key ``"use_elastic_search": False`` in the ``QUESTION_ANSWERER_CONFIG`` as follows.
+Search operations of QuestionAnswerer are by default backed by Elasticsearch. While it is recommended to use the Elasticsearch based for QuestionAnswerer, it might not always be feasible to use it due to some or the other constraints. In such cases, one can easily use a non-Elasticsearch backend by setting `"model_type": "native"` in the ``QUESTION_ANSWERER_CONFIG`` as follows.
```

Comment:
```suggestion
Search operations of QuestionAnswerer are by default backed by Elasticsearch. While it is recommended to use the Elasticsearch based QuestionAnswerer, it might not always be feasible to use it due to various constraints. In such cases, one can easily disable Elasticsearch by setting `"model_type": "native"` in the ``QUESTION_ANSWERER_CONFIG`` as follows.
```